python/fusion_engine_client/messages/device.py

```diff
@@ -0,0 +1,47 @@
+import struct
+
+from construct import (Struct, Int64ul, Int16ul, Int8ul, Int16sl, Padding, this, Bytes, PaddedString)
+
+from ..utils.construct_utils import AutoEnum, construct_message_to_string
+from ..utils.enum_utils import IntEnum
+from .defs import *
+
+class SystemStatusMessage(MessagePayload):
+    """!
+    @brief System status elements.
+    """
+    MESSAGE_TYPE = MessageType.SYSTEM_STATUS
+    MESSAGE_VERSION = 0
+
+    INVALID_TEMPERATURE = -32768
+
+    SystemStatusMessageConstruct = Struct(
+        "p1_time" / TimestampConstruct,
+        "gnss_temperature" / Int16sl,
+        Padding(131),
+    )
+
+    def __init__(self):
+        self.p1_time = Timestamp()
+        self.gnss_temperature = SystemStatusMessage.INVALID_TEMPERATURE
```

Comment:
In Python, we prefer to store members in standard units (`gnss_temperature_degc`), and do the conversion in the pack and unpack functions.